esrally/driver/runner.py

```diff
@@ -142,6 +142,28 @@ async def __call__(self, *args):
     async def __aexit__(self, exc_type, exc_val, exc_tb):
         return False
 
+    def _default_kw_params(self, params):
+        # map of API kwargs to Rally config parameters
+        kw_dict = {
+            "body": "body",
+            "index": "index",
+            "opaque_id": "opaque-id",
```

Comment:
because this exists in the queryparams decorator applied to all API calls that we use, it was trivial to add it here for handling.  the other thing I want to add is `headers`, which technically collides with the opaque_id but precedence is handled here: https://github.com/elastic/elasticsearch-py/blob/master/elasticsearch/client/utils.py#L140